Src/Support/GoogleApis.Tests/Apis/Requests/Parameters/ParameterUtilsTest.cs

```diff
@@ -0,0 +1,74 @@
+﻿/*
+Copyright 2016 Google Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+using System.Collections.Generic;
+
+using NUnit.Framework;
+
+using Google.Apis.Util;
+using Google.Apis.Requests;
+using Google.Apis.Requests.Parameters;
+
+namespace Google.Apis.Tests.Apis.Requests
+{
+    [TestFixture]
+    public class ParameterUtilsTest
+    {
+        private class TestRequestUrl
+        {
+            [Google.Apis.Util.RequestParameterAttribute("first_query_param", Google.Apis.Util.RequestParameterType.Query)]
```

Comment:
I removed it off the second parameter so that it's testing a query parameter with the name provided and a value provided, the second one with just a value provided, and the custom parameters. Is that what you were looking for? Otherwise I'm not sure where else to remove a name.